src/pages/playground/PlaygroundTabs.tsx

```diff
@@ -64,7 +39,10 @@ export const makeRemoteExecutionTabFrom = (
   id: SideContentType.remoteExecution
 });
 
-export const makeSubstVisualizerTabFrom = (output: InterpreterOutput[]): SideContentTab => {
+export const makeSubstVisualizerTabFrom = (
```

Comment:
Should we move all of these tabs under `PlaygroundTabs` into the respective `SideContentXXX` (and just delete the `PlaygroundTabs` file)? Since the `makeXXX` are being used in not just the playground (even if they are, the functions are technically workspace-agnostic).

Reply:
I think it might be worth looking to as the rest of the workspaces get refactored, but for now I'm trying to minimize the number of things that I need to change.